doc/functions/cva.rst

```diff
@@ -0,0 +1,172 @@
+``cva``
+=======
+
+`CVA (Class Variant Authority)`_ is a concept from the JavaScript world and used
+by the well-known `shadcn/ui`_.
+CVA allows you to display a component with different variants (color, size, etc.),
+to create highly reusable and customizable templates. This function work best by being combined with
+`TwigComponent`_
```

Comment:
I'd maybe mention this later in the page, as TwigComponent is --despite its name-- a Symfony package.